hw/top_earlgrey/BUILD

```diff
@@ -240,16 +240,7 @@ sim_dv(
 sim_dv(
     name = "sim_dv",
     testonly = True,
-    args = [
-        "{dvsim_config}",
-    ],
     base = ":sim_dv_base",
-    data = [
-        "//hw/top_earlgrey/dv:chip_sim_cfg.hjson",
-    ],
     exec_env = "sim_dv",
-    param = {
-        "dvsim_config": "$(location //hw/top_earlgrey/dv:chip_sim_cfg.hjson)",
-    },
```

Comment:
I had tried overriding with the `@//` prefix and didn't work, but it does work if I add it here. I will leave this removed, but can add it back if you think it would be helpful in the future.